### PR TITLE
[Hub Generated] Review request for Microsoft.CostManagement to add version 2019-03-01-preview

### PR DESCRIPTION
**Pull request by ariehf**
This PR contains the following new resourceTypes for CostManagement:
•	cloudConnectors
•	externalBillingAccounts
•	externalSubscriptions
•	getEntities

If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/ariehf/azure-rest-api-specs/dev-cost-management-Microsoft.CostManagement-2019-03-01-preview).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by mozehgir** [comment]
Hi @ariehf I though the plan was to create an empty folder first? I will setup a meeting with Samer Monday early morning PST to chat with you

**Comment by praries880** [comment]
@ariehf kindly refer to the email response WRT this PR. Kindly fix the build breaks in the PR before we can proceed with the review. I have requested an  ARM review as well for the changes.

**Comment by ariehf** [comment]
This PR contains the following new resourceTypes for CostManagement:
•	cloudConnectors 
•	externalBillingAccounts
•	externalSubscriptions
•	getEntities

**Comment by ariehf** [comment]
The cloudConnectors in this PR is a refactoring of the connectors resource type from 2018-08-01-preview.
The following refactoring was done from the old connectors:
•	Rename connectors->cloudConnectors
•	Move from subscription level to tenant Level
•	Remove subscriptions & resourceGroup
•	Remove "location"
•	The following within Connector properties:
•	Add "subscriptionId", "billingModel", "daysTrialRemaining"
•	Rename collection->collectionInfo
•	Added collectionInfo/errorInnerMessage
•	Rename lastRun to lastChecked

**Review by KrisBash — APPROVED** [review]
LGTM - thanks for the review meeting

**Review by praries880 — APPROVED** [review]

File: specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json [hunk 1]
```diff
@@ -0,0 +1,2872 @@
...
+          "nextLinkName": null
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{managementGroupId}/providers/Microsoft.CostManagement/Query": {
+      "post": {
+        "tags": [
+          "Query"
+        ],
+        "operationId": "Query_UsageByManagmentGroup",
```
Comment (nschonni) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json]: Managment -> Management

File: specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json [hunk 2]
```diff
@@ -0,0 +1,2872 @@
...
+          "nextLinkName": null
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{managementGroupId}/providers/Microsoft.CostManagement/Forecast": {
+      "post": {
+        "tags": [
+          "Forecast"
+        ],
+        "operationId": "Forecast_UsageByManagmentGroup",
```
Comment (nschonni) [inline specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json]: Managment -> Management

**Pull request merged**